spec/ref_arch_setup/bolt_helper_spec.rb

```diff
@@ -130,10 +131,31 @@
       "#{RefArchSetup::RAS_MODULE_PATH} --nodes #{nodes}"
     end
 
+    context "when a modulepath is specified" do
+      expected_output = "All Good"
+      expected_status = 0
+      modulepath = "./my_modules"
+
+      it "uses the specified value" do
+        expected_command = "bolt task run #{task} VAR1=1 VAR2=2 --modulepath "\
+          "#{modulepath} --nodes #{nodes}"
+
+        expect(RefArchSetup::BoltHelper).to receive(:params_to_string) \
+          .with(params).and_return(params_str)
+        expect(RefArchSetup::BoltHelper).to receive(:`)\
+          .with(expected_command).and_return(expected_output)
+        `(exit #{expected_status})`
+        expect($?).to receive(:success?).and_return(true) # rubocop:disable Style/SpecialGlobalVars
+        allow(RefArchSetup::BoltHelper).to receive(:puts)
+        expect(RefArchSetup::BoltHelper.run_task_with_bolt(task, params, nodes, modulepath))
+          .to eq(true)
+      end
+    end
+
     context "when bolt works and returns true" do
       expected_output = "All Good"
       expected_status = 0
-      it "returns true" do
+      it "uses the default modulepath and returns true" do
```

Comment:
I remember Eric saying having "and" in the "it" title is bad because you don't know which thing failed if the test fails.  maybe create a single test to verify that it uses the default and run it first?

Reply:
Updated to include a separate case for 'uses the default modulepath'